targeteditor/src/main/scala/explore/targeteditor/plotPeriod.scala

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.targeteditor
+
+import java.time.Duration
+import java.time.Instant
+import java.time.LocalDate
+
+import gem.Semester
+import gem.enum.Site
+import gem.math.LocalObservingNight
+import gem.math.ObservingNight
+
+sealed trait PlotPeriod {
+  def start(site: Site): Instant
```

Comment:
Does this mean we could create a PlotPeriod with start on GN and end on GS?

Reply:
The PlotPeriod is independent of the site. You can ask it its start and end instants in a Site. For example: for a NightPlot, the idea is to get the period from dusk till dawn, which yields different start and end times in each site. This is not working yet since we have to migrate the code to compute sunset and sunrise.